internal/utils/findings/findings.go

```diff
@@ -0,0 +1,63 @@
+package findings
+
+import (
+	"slices"
+
+	"github.com/snyk/go-application-framework/pkg/local_workflows/local_models"
+	"github.com/snyk/go-application-framework/pkg/utils"
+)
+
+type FindingsFilterFunc func(local_models.FindingResource) bool
+
+func filterSeverityASC(original []string, severityMinLevel string) []string {
+	if severityMinLevel == "" {
+		return original
+	}
+
+	minLevelPointer := slices.Index(original, severityMinLevel)
+
+	if minLevelPointer >= 0 {
+		return original[minLevelPointer:]
+	}
+
+	return original
+}
+
+func GetSeverityThresholdFilter(severityThreshold string, severityOrder []string) FindingsFilterFunc {
+	return func(finding local_models.FindingResource) bool {
+		allowed_severities := filterSeverityASC(severityOrder, severityThreshold)
```

Comment:
Suggestion: This could be moved out of the returned func to be a bit more efficient. This will avoid re-filtering

Reply:
done, thank you